lib/repl.js

```diff
@@ -65,16 +66,23 @@ export default class REPL {
       .replace(/ \(bound at.*/g, "")
   }
 
-  tidalSendExpression(expression) {
+  tidalSendExpression(expression, range) {
     this.tidalSendLine(':{');
 
     expression.split('\n')
-      .forEach(line => this.tidalSendLine(line));
+      .forEach((line, index) => {
+        this.tidalSendLine(line, range ? range.start.row + index : undefined)
+      });
 
     this.tidalSendLine(':}');
   }
 
-  tidalSendLine(command) {
+  tidalSendLine(command, lineNumber) {
+    if (!this.editors.currentIsTidal()) return;
```

Comment:
this should be necessary as there's no way to send a tidal line on a non-tidal editor. Am I correct?